### PR TITLE
Make bundle size smaller (replaced uniqBy and updated react-hook-form)

### PR DESCRIPTION
**Pull request by svenjens**

### CODE REVIEW

**Comment by robinpiets** [comment]
Nice! `lodash.uniqby` is also in the `/apps/graphql` ;)

File: apps/client/src/utils/index.js [hunk 1]
```diff
@@ -25,3 +25,26 @@ export const isObject = (val) => typeof val === "object" && val !== null;
 // Filters
 export const uniqueFilter = (value, index, self) =>
   self.indexOf(value) === index;
+
+export const uniqBy = (arr, predicate) => {
```
Comment (robinpiets) [inline apps/client/src/utils/index.js:31]: Can you add a comment?

**Comment by svenjens** [comment]
> Nice! `lodash.uniqby` is also in the `/apps/graphql` ;)

But that isn't in the client bundle :-) So it isn't making our build bigger.

File: apps/client/package.json [hunk 2]
```diff
@@ -25,14 +25,13 @@
...
     "lodash.isstring": "^4.0.1",
-    "lodash.uniqby": "^4.7.0",
     "named-urls": "^2.0.0",
     "react": "^16.13.1",
     "react-app-polyfill": "^1.0.6",
     "react-device-detect": "^1.12.1",
     "react-dom": "^16.13.1",
     "react-helmet": "^6.1.0",
-    "react-hook-form": "^5.7.2",
+    "react-hook-form": "^6.0.0",
```
Comment (robinpiets) [inline apps/client/package.json:34]: Why is this in the uniqBy pr?

**Review by robinpiets — CHANGES_REQUESTED** [review]

File: apps/client/package.json (hunk 2 above)
Comment (svenjens) [reply apps/client/package.json:34]: Because this is a pr to make the bundle sizes smaller. This makes the bundles size a lot smaller.

Comment (robinpiets) [reply apps/client/package.json:34]: Cool!

**Review by robinpiets — APPROVED** [review]
Please change the comment and Squash and merge

**Pull request merged**